ops/pipeline/build-test-jvm-packages.sh

```diff
@@ -21,8 +23,10 @@ do
   fi
 done
 
+CONTAINER_TAG=${DOCKER_REGISTRY_URL}/xgb-ci.jvm:main
```

Comment:
I would probably have called this `IMAGE_URI` instead of `CONTAINER_TAG`, since this is the fully-qualified identifier for an image. The "tag" in this case is just `:main`.

In general, I recommend following this type of naming for the variables involved here:

* `IMAGE_REGISTRY` = the place to pull the images from, like
  - *(e.g. `docker.io`, `quay.io`, `ghcr.io`, an ECR URL)*
* `IMAGE_REPO` = the name of the image, called a "repository" on some registries
  - *(e.g. `xgb-ci.jvm`, `dask/dask-notebook`)*
* `IMAGE_TAG` = ID that separates images from the same registry and repo
  - *(e.g. `2024.10`, `main`, `stable`)*
* `IMAGE_URI` = fully-qualified identifier of the form `${IMAGE_REGISTRY}/${IMAGE_REPO}:${IMAGE_TAG}`
  - *(e.g. `quay.io/pypa/manylinux2014_aarch64:latest`)*

I know this might seem minor and pedantic, but being precise about these terms (and about the difference between "container" and "image") is helpful for others reading these scripts and config files.